app/src/main/java/com/example/keniphhomemaintenance/MainActivity.kt

```diff
@@ -1,38 +1,32 @@
 package com.example.keniphhomemaintenance
 
+import android.content.res.Configuration
 import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
-import androidx.compose.material.MaterialTheme
-import androidx.compose.material.Surface
-import androidx.compose.material.Text
+import androidx.compose.material.*
 import androidx.compose.runtime.Composable
```

Comment:
a bunch of these imports are unused, clean them up using ctrl+alt+O (the letter oh, not the number zero)